packages/fx-core/resource/package.nls.json

```diff
@@ -194,5 +194,72 @@
+  "core.boot.BotResourceExists": "Bot resource already existed on",
+  "core.boot.SkipCreateBotResource": "skip creating Bot resource.",
+  "core.boot.WorkingDirMissing": "Working directory is missing.",
+  "core.boot.FailRetrieveAzureCredentials": "Failed to retrieve Azure credentials.",
+  "core.boot.LoginToAzure": "Login to Azure.",
+  "core.boot.SkipDeployment": "Skipping deployment: no updates found.",
+  "core.boot.ScaffoldingBot": "Scaffolding bot.",
+  "core.boot.ScaffoldingBotSuccess": "Successfully scaffolded bot.",
+  "core.boot.PreProvisionBot": "Pre-provisioning bot.",
+  "core.boot.ProvisionBot": "Provisioning bot.",
+  "core.boot.PreDeployingBot": "Pre-deploying bot.",
+  "core.boot.DeployingBot": "Deploying bot.",
+  "core.boot.DeployingBotSuccess": "Successfully deployed bot.",
+  "core.boot.GeneratingBotARMTemplates": "Generating ARM templates of bot.",
+  "core.boot.GeneratingBotArmTemplatesSuccess": "Successfully generating ARM templates of bot.",
+  "core.boot.UpdateBotARMTemplates": "Updating ARM templates of bot.",
+  "core.boot.UpdateBotARMTemplatesSuccess": "Successfully updating ARM templates of bot.",
+  "core.boot.GetBotAADSuccess": "Successfully get existing bot AAD app credential.",
+  "core.boot.CreateBotAADSuccess": "Successfully created bot AAD app.",
+  "core.boot.UpdateAzureWebAppSetting": "Updating azure web app's app settings.",
+  "core.boot.ProvisionBotReg": "Provisioning bot registration.",
+  "core.boot.ProvisionBotRegSuccess": "Successfully provisioned bot registration.",
+  "core.boot.SubscriptionNoReg": "The subscription didn't register to use namespace 'Microsoft.BotService'.",
+  "core.boot.RetryCurrent": "Please retry the current step.",
+  "core.boot.RegSubscription": "Please register your subscription to use namespace 'Microsoft.BotService'.",
+  "core.boot.RecoverConfig": "Please recover the config value.",
+  "core.boot.RecreateProject": "Please recreate the project.",
+  "core.boot.CheckCommandOutput": "Please check the command output and try to fix it.",
+  "core.boot.DeleteExistsAzureBotChannelReg": "Please delete existing azure bot channel registrations.",
+  "core.boot.DelAzureBotChannel": "If azure account is switched, don't forget to delete azure bot channel registration under the previous account.",
+  "core.boot.CheckLogAndFix": "Please check log in output channel and try to fix this issue.",
+  "core.boot.Register": "Register",
+  "core.boot.ResourceProviderManually": "resource provider for your subscription manually.",
+  "core.boot.CheckPath": "Please check whether the path",
+  "core.boot.PathWriteAccess": "exists and you have write access to it, if no, please move the project to a valid path.",
+  "core.boot.AzureWebApp": "Azure Web App",
+  "core.boot.AzureBotServiceChannelReg": "Azure Bot Service channel registration",
+  "core.boot.TeamsChannel": "Teams channel",
+  "core.boot.AadApp": "AAD app",
+  "core.boot.AadClientSec": "AAD client secret",
```

Comment:
I think it's better to user `Secret` here. `Sec` might refer to other words such as 'second'.

Reply:
fixed